docs/users/customization-extendibility.md

```diff
@@ -79,5 +79,14 @@ services:
       - solr:$DDEV_HOSTNAME
 ```
 
+## Providing custom nginx configuration
```

Comment:
I don't think that this can be expressed as well in a paragraph as in an example. We probably need an examples directory here with a super-simple conf in it.